ext/libxml/config.w32

```diff
@@ -10,7 +10,7 @@ if (PHP_LIBXML == "yes") {
 			ADD_EXTENSION_DEP('libxml', 'iconv')) {
 
 		EXTENSION("libxml", "libxml.c mime_sniff.c", false /* never shared */, "/DZEND_ENABLE_STATIC_TSRMLS_CACHE=1");
-		AC_DEFINE("HAVE_LIBXML", 1, "LibXML support");
+		AC_DEFINE("HAVE_LIBXML", 1, "Define to 1 if libxml2 library is available.");
```

Comment:
HAVE_LIBXML means we have the libxml extension (and therefore indirectly we also know we have the libxml library), so the description is strictly speaking subtly wrong.

Reply:
Ok, I'll recheck the extensions out there. Yes, this definition is better done in the libxml extension only and not in the setup macro... A bit of an inconsistency that wasn't fixed by now. Because, the PHP_SETUP_LIBXML can be used by PECL extensions and then this HAVE_LIBXML gets defined... And theoretically, such PECL EXTENSION wouldn't even depend on libxml extension.

Reply:
Right, I see, a bit of a messy history around this. Also doesn't help that the extension has the same name as the library of course...

Reply:
I think all is good here to remove the redundant HAVE_LIBXML definition from the PHP_SETUP M4 macro. I haven't found any extension that uses this symbol. So I've only added a simple note in the UPGRADING.INTERNALS file.